src/extensions/default/bramble/lib/PostMessageTransport.js

```diff
@@ -105,6 +106,11 @@ define(function (require, exports, module) {
     function start(){
         window.addEventListener("message", _listener);
 
+        var autoUpdate = BrambleStartupState.ui("autoUpdate");
+        if(typeof autoUpdate === "boolean") {
```

Comment:
is there a possibility that this is not a boolean? What are the values that it can take?